fdbserver/team/TestBase.actor.cpp

```diff
@@ -0,0 +1,749 @@
+#include <cinttypes>
+#include <deque>
+#include <iomanip>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "fdbclient/CommitTransaction.h"
+#include "fdbclient/FDBTypes.h"
+#include "fdbrpc/fdbrpc.h"
+#include "fdbserver/Knobs.h"
+#include "flow/crc32c.h"
+#include "flow/DeterministicRandom.h"
+#include "flow/UnitTest.h"
+
+#include "flow/actorcompiler.h"
+
+static const int NUM_TEAMS = 12;
+static const int NUM_TLOG_PROCESSES = 2;
+static const int NUM_STORAGE_SERVER_PROCESSES = 2;
+
+namespace Team {
+
+struct TeamID {
+	constexpr static FileIdentifier file_identifier = 345787;
+
+	int32_t teamID;
```

Comment:
`TeamID` is an `UID`, see definition here: https://github.com/jzhou77/foundationdb/blob/49a43dd12cd274d517182f952bfd405be2f6ec98/fdbserver/DataDistribution.actor.cpp#L167

Reply:
Is it necessary to be a class?

Reply:
For the teamID, I think the issue is that the old Tag is a class, with locality information.